com.unity.perception/CHANGELOG.md

```diff
@@ -5,39 +5,51 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [0.6.0-preview.1] - 2020-12-03
```

Comment:
It's probably smart to add in the boilerplate Unreleased area and all of the sub-headers just to get it ready for people to add their changes to it.

Reply:
done